src/js/lib/facebook/fb_login_script.js

```diff
@@ -0,0 +1,26 @@
+export default function fb_login_init(loginSuccessHandler) {
+  window.fbAsyncInit = function() {
+    FB.init({
+      appId      : '1019149808185946',
+      xfbml      : true,
+      version    : 'v2.6'
+    });
+    FB.AppEvents.logPageView();
+
+    if(!loginSuccessHandler) return;
+
+    FB.getLoginStatus(function(response) {
+      console.log('the response:', response)
+      loginSuccessHandler(response)
+    });
+  };
+
+  (function(d, s, id){
+    var js, fjs = d.getElementsByTagName(s)[0];
```

Comment:
@andela-iamadi Hound caught you :). Why can't we use `let` to declare the variables?